styles.css

```diff
@@ -0,0 +1,78 @@
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    font-family: 'Georgia', serif;
+}
+body{
+    background: #F1879E;
+}
+.app{
+    background: #EECFCC;
+    width: 90%;
+    max-width: 600px;
+    margin: 100px auto 0;
+    border-radius: 40px;
+    padding: 30px;
+}
+.app h1{
+    font-size: 25px;
+    color: #F1879E;
+    font-weight: 600;
+    border-bottom: 1px solid white;
+    padding-bottom: 30px;
+    text-align: center;
+}
+.quiz{
+    padding: 20px 0;
+}
+.quiz h2{
+    font-size: 24px;
+    color: #F1879E;
+    font-weight: 600;
+    text-align: center;
+}
+.btn{
+    background: #F1879E;
+    color: #F5ECEB;
+    font-weight: 500;
+    width: 100%;
+    border: 2px solid #F5ECEB;
+    padding: 10px;
+    margin: 10px 0;
+    text-align: left;
+    border-radius: 5px;
+    cursor: pointer;
+    transition: all 0.3s;
+}
+.btn:hover:not([disabled]){
+    background: #F5ECEB;
```

Comment:
I can notice you use multipile times this color. And maybe other...
There is something that is called variables in CSS as well. Of corse the reason is the same as for using constans variables for ANSWER_HTML_TAG or ANSWER_CLASSES.

[More here.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

```css
/*TO ZAWSZE NA SAMEJ GORZE POWINNO BYC... BO TO TAK JAK ZMIENNE GLOBALNE W JS*/
:root {
    --main-bg-color: #F5ECEB;
}

.btn:hover:not([disabled]){
    background: var(--main-bg-color);
    /*reszta...*/
}
```

Reply:
Off course add more variables if any color or sizes repeat in the code...

Reply:
Nie rozumiem :3

Reply:
zrobione